sdk/python/kfp_tekton/tekton.py

```diff
@@ -31,34 +32,64 @@ class AnySequencer(ContainerOp):
                 because the pipeline will generate a unique new name in case of conflicts.
 
         any: List of `Conditional` containerOps that deploy together with the `main`
-                containerOp.
+                containerOp, or the condtion that must meet to continue.
     """
     def __init__(self,
-                 any: List[dsl.ContainerOp],
+                 any: Iterable[Union[dsl.ContainerOp, ConditionOperator]],
                  name: str = None,):
 
         tasks_list = []
+        condition_list = []
         for cop in any:
-            cop_name = sanitize_k8s_name(cop.name)
-            tasks_list.append(cop_name)
+            if isinstance(cop, dsl.ContainerOp):
+                cop_name = sanitize_k8s_name(cop.name)
+                tasks_list.append(cop_name)
+            elif isinstance(cop, ConditionOperator):
+                condition_list.append(cop)
+
         task_list_str = ",".join(tasks_list)
+        arguments = [
+                    "--namespace",
+                    "$(context.pipelineRun.namespace)",
+                    "--prName",
+                    "$(context.pipelineRun.name)",
+                    "--taskList",
+                    task_list_str
+                ]
+        conditonArgs = processConditionArgs(condition_list)
+        arguments.extend(conditonArgs)
 
         super().__init__(
             name=name,
             image="dspipelines/any-sequencer:latest",
             command="any-taskrun",
-            arguments=[
-                        "-namespace",
-                        "$(context.pipelineRun.namespace)",
-                        "-prName",
-                        "$(context.pipelineRun.name)",
-                        "-taskList",
-                        task_list_str
-                    ],
+            arguments=arguments,
         )
 
 
-def after_any(any: List[dsl.ContainerOp], name: str = None):
+def processOperand(operand) -> str:
+    if isinstance(operand, dsl.PipelineParam):
+        return "results_" + operand.op_name + "_" + operand.name
```

Comment:
can here to ensure the op_name is one of task in the pipelines? I meant if the task is not in the pipelines, not sure is there any problem?

Reply:
PipelineParam's op_name should be one of task in the pipelines, here we need assert the operand is either int or string or PipelineParam like I mentioned before.

Reply:
@pugangxa Why does the above use `"_"`? It seems to me that `"_"` is allowed for parameter names in Tekton, so wouldn't some forbidden character, like `"."` be better?

Reply:
Yes this is one of the places need fix I mentioned in comment. It's a little tricky because some characters that's allowed in Tekton but not allowed for goevaluate, and some in goevaluate not allowed in Tekton. I don't want to translate between goevaluate and Tekton so maybe use the format like ```taskName_output_outputName_end``` which is easy to parse and allowed by both.

Reply:
Just checked that "_" is not allowed  for parameter names in Tekton, in kfp-tekton all "_" will be replaced with "-", so we can still use it but must process '-' in goevaluate since it's not allowed in goevaluate